diff --git bundles/org.eclipse.jface/.settings/org.eclipse.jdt.core.prefs bundles/org.eclipse.jface/.settings/org.eclipse.jdt.core.prefs
index 347e248..ab4e077 100644
--- bundles/org.eclipse.jface/.settings/org.eclipse.jdt.core.prefs
+++ bundles/org.eclipse.jface/.settings/org.eclipse.jdt.core.prefs
@@ -7,9 +7,9 @@ org.eclipse.jdt.core.circularClasspath=error
 org.eclipse.jdt.core.classpath.exclusionPatterns=enabled
 org.eclipse.jdt.core.classpath.multipleOutputLocations=enabled
 org.eclipse.jdt.core.compiler.codegen.inlineJsrBytecode=enabled
-org.eclipse.jdt.core.compiler.codegen.targetPlatform=1.2
+org.eclipse.jdt.core.compiler.codegen.targetPlatform=1.5
 org.eclipse.jdt.core.compiler.codegen.unusedLocal=preserve
-org.eclipse.jdt.core.compiler.compliance=1.4
+org.eclipse.jdt.core.compiler.compliance=1.5
 org.eclipse.jdt.core.compiler.debug.lineNumber=generate
 org.eclipse.jdt.core.compiler.debug.localVariable=generate
 org.eclipse.jdt.core.compiler.debug.sourceFile=generate
@@ -67,6 +67,6 @@ org.eclipse.jdt.core.compiler.problem.unusedParameterWhenImplementingAbstract=di
 org.eclipse.jdt.core.compiler.problem.unusedParameterWhenOverridingConcrete=disabled
 org.eclipse.jdt.core.compiler.problem.unusedPrivateMember=error
 org.eclipse.jdt.core.compiler.problem.varargsArgumentNeedCast=warning
-org.eclipse.jdt.core.compiler.source=1.3
+org.eclipse.jdt.core.compiler.source=1.5
 org.eclipse.jdt.core.incompatibleJDKLevel=ignore
 org.eclipse.jdt.core.incompleteClasspath=error
diff --git bundles/org.eclipse.jface/pom.xml bundles/org.eclipse.jface/pom.xml
index af40736..6c5fdf2 100644
--- bundles/org.eclipse.jface/pom.xml
+++ bundles/org.eclipse.jface/pom.xml
@@ -21,16 +21,5 @@
   <artifactId>org.eclipse.jface</artifactId>
   <version>3.9.0-SNAPSHOT</version>
   <packaging>eclipse-plugin</packaging>
-  <build>
-    <plugins>
-      <plugin>
-        <groupId>org.eclipse.tycho</groupId>
-        <artifactId>target-platform-configuration</artifactId>
-        <configuration>
-          <!-- juno version of this bundle was aparently compiled against jdk 1.4 librariesF-->
-          <executionEnvironment>J2SE-1.4</executionEnvironment>
-        </configuration>
-      </plugin>
-    </plugins>
-  </build>
+
 </project>
