diff --git bundles/org.eclipse.jface.databinding/META-INF/MANIFEST.MF bundles/org.eclipse.jface.databinding/META-INF/MANIFEST.MF
index 31cceb7..452f1d4 100644
--- bundles/org.eclipse.jface.databinding/META-INF/MANIFEST.MF
+++ bundles/org.eclipse.jface.databinding/META-INF/MANIFEST.MF
@@ -24,5 +24,4 @@ Require-Bundle: org.eclipse.swt;bundle-version="[3.2.0,4.0.0)",
  org.eclipse.core.databinding.observable;bundle-version="[1.3.0,2.0.0)",
  org.eclipse.core.databinding.property;bundle-version="[1.3.0,2.0.0)",
  org.eclipse.core.databinding;bundle-version="[1.2.0,2.0.0)"
-Bundle-RequiredExecutionEnvironment: CDC-1.0/Foundation-1.0,
- J2SE-1.3
+Bundle-RequiredExecutionEnvironment: J2SE-1.5
diff --git bundles/org.eclipse.ui/META-INF/MANIFEST.MF bundles/org.eclipse.ui/META-INF/MANIFEST.MF
index 841b4ed..ea9fb0f 100644
--- bundles/org.eclipse.ui/META-INF/MANIFEST.MF
+++ bundles/org.eclipse.ui/META-INF/MANIFEST.MF
@@ -14,5 +14,4 @@ Require-Bundle: org.eclipse.core.runtime;bundle-version="[3.2.0,4.0.0)",
  org.eclipse.jface;bundle-version="[3.9.0,4.0.0)";visibility:=reexport,
  org.eclipse.ui.workbench;bundle-version="[3.105.0,4.0.0)";visibility:=reexport,
  org.eclipse.core.expressions;bundle-version="[3.4.0,4.0.0)"
-Bundle-RequiredExecutionEnvironment: CDC-1.0/Foundation-1.0,
- J2SE-1.3
+Bundle-RequiredExecutionEnvironment: J2SE-1.5
